docs/src/main/tut/faq.md

```diff
@@ -16,7 +16,7 @@ section: "faq"
  * [What does `macro Ops` do? What is `cats.macros.Ops`?](#machinist)
  * [How can I help?](#contributing)
 
-## What imports do I need?<a id="what-imports" href="#what-imports"></a>
+## <a id="what-imports" href="#what-imports"></a>What imports do I need?
```

Comment:
My own CSS skills aren't particularly advanced, but if I get some time I'll come back to these links and see if I can find a better way.